extensions/toggl-track/package.json

```
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "toggl-track",
+  "title": "Toggl Track",
+  "description": "Start and stop time entries with Toggl Track",
+  "icon": "command-icon.png",
+  "author": "franzwilhelm",
+  "license": "MIT",
+  "owner": "franzwilhelm",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Toggl Track",
```

```suggestion
      "title": "Time Tracking",
      "subtitle": "Toggl Track",
```

---

Actually, I would follow the <Verb> <Norm> structure. Also just Toggl as subtitle should be enough, right?
```suggestion
      "title": "Track Time",
      "subtitle": "Toggl",
```

---

Agreed. Fixed